### ISSUE TITLE
System unavailable: Xvfb seems nonresponsive on test-osuosl-ubuntu1804-ppc64le-2

### ISSUE THREAD

**Issue by adamfarley** [adoptium/infrastructure#3171]
- Please put the system name in the title of this issue.

- Link to any log file showing the problem: https://ci.adoptium.net/job/Test_openjdk22_hs_extended.system_ppc64le_linux/3/

- Please describe the issue:

```
Exception: java.lang.IllegalStateException: No display name received from Xvfb within 30 seconds
```

This has been happening for while on this platform, though I cannot prove that it was definitely this machine.

**Comment by adamfarley** [comment]
Seen twice more:
[test-osuosl-ubuntu2004-ppc64le-1](https://ci.adoptium.net/computer/test-osuosl-ubuntu2004-ppc64le-1): https://ci.adoptium.net/job/Test_openjdk11_hs_sanity.system_ppc64le_linux/826/console
[test-docker-ubuntu1804-ppc64le-1](https://ci.adoptium.net/computer/test-docker-ubuntu1804-ppc64le-1): https://ci.adoptium.net/job/Test_openjdk11_hs_extended.openjdk_ppc64le_linux_testList_0/118/console

**Comment by adamfarley** [comment]
Likely a duplicate of https://github.com/adoptium/aqa-tests/issues/4930

**Comment by steelhead31** [comment]
A good number of these issues are on a single dockerhost, dockerhost-osuosl-ubuntu2004-ppc64le-1  , this host has 3 containers **test-docker-fedora39-ppc64le-1 , test-docker-ubuntu2004-ppc64le-1 & test-docker-ubuntu2204-ppc64le-3**

**Comment by steelhead31** [comment]
I'll test by stopping a single container ( **test-docker-ubuntu2004-ppc64le-1 )**, to see if performance is improved.  
  
Rerun in grinder on **test-docker-ubuntu2204-ppc64le-3** with above container & node stopped https://ci.adoptium.net/job/Grinder/10788/

**Comment by steelhead31** [comment]
I've identified and corrected a permissions issue on **test-docker-ubuntu2204-ppc64le-3** ( /tmp/.X11-unix was owned by Jenkins and it must be owned by root )..  new grinder run is here...  #3171 which seems to be proceeding correctly. Once this run is finished, I'll test it on the other docker containers.

**Comment by steelhead31** [comment]
Running same tests on **test-docker-ubuntu2004-ppc64le-1 here :** https://ci.adoptium.net/job/Grinder/10792/

**Comment by steelhead31** [comment]
And passed on the fedora container here. Im going to close this, please let me know if this recurrs and what machine, as I believe the underlying issue has been resolved.
